core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -10407,4 +10407,30 @@ private void checkLoptOptimizeJoinRule(LoptOptimizeJoinRule rule) {
         .withRule(CoreRules.JOIN_CONDITION_PUSH)
         .check();
   }
+
+  /** Test case of
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6973">[CALCITE-6973]
+   * Add rule for convert Minus to Filter</a>. */
+  @Test void testMinusToFilterRule() {
+    final String sql = "SELECT mgr, comm FROM emp WHERE mgr = 12\n"
+        + "EXCEPT\n"
+        + "SELECT mgr, comm FROM emp WHERE comm = 5\n";
+    sql(sql)
+        .withPreRule(CoreRules.PROJECT_FILTER_TRANSPOSE)
+        .withRule(CoreRules.MINUS_TO_FILTER)
+        .check();
+  }
+
```

Comment:
Can we add a test case involving `FILTER_MERGE` rule too just to check that we can cover equivalent transformations as the previous version?

Reply:
In my opinion, it may not be necessary to add this case. This case may only show whether the filter is merged, which has little to do with this PR.